src/main/java/it/gov/pagopa/payment/exception/TransactionSynchronousExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.payment.exception;
+
+import it.gov.pagopa.payment.dto.qrcode.AuthPaymentDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+@Slf4j
+public class TransactionSynchronousExceptionHandler {
+
+  @ExceptionHandler(RuntimeException.class)
+  protected ResponseEntity<AuthPaymentDTO> handleException(RuntimeException error){
+    AuthPaymentDTO response;
+    HttpStatus httpStatus;
+    if (error instanceof TransactionSynchronousException transactionSynchronousException) {
```

Comment:
Let's specify the particular handled exception.
This way the ErrorManager will take care of all the other Exceptions
```suggestion
  @ExceptionHandler(TransactionSynchronousException .class)
  protected ResponseEntity<AuthPaymentDTO> handleException(TransactionSynchronousException transactionSynchronousException){
    AuthPaymentDTO response;
    HttpStatus httpStatus;
```